libraries/AP_Landing/AP_Landing_Deepstall.h

```diff
@@ -44,13 +44,13 @@ class AP_Landing_Deepstall
 
     // deepstall members
     enum deepstall_stage {
```

Comment:
How about an enum class instead, if you're changing everything anyway?

Reply:
Nope. You can convince me to just use the MAVLink enum though, since I wanted a 1:1 mapping anyways :)